src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
+
+use crate::prelude::*;
+
+use super::GLYPH_WIDTH;
```

Comment:
There are no constants to set top and left margin for the breadcrumbs - such constants should be exposed.